Shared/app-service.json

```diff
@@ -64,7 +71,8 @@
             },
             "resources": [
                 {
-                    "name": "staging",
+                    "condition": "[parameters('includeDeploymentSlot')]",
+                    "name": "requiresDeploymentSlot",
```

Comment:
I believe this "name" is currently set in the deployments, have those been changed as well?